local/blockchain.go

```diff
@@ -98,6 +98,10 @@ func (ln *localNetwork) getNode() node.Node {
 }
 
 func (ln *localNetwork) getMinValidatorWeight() uint64 {
+	// special case for custom networks with networkID not in the list with passed genesis filepath
+	if ln.genesisPath != "" {
+		return genesis.FujiParams.MinValidatorStake // we treat custom network as fuji for now
```

Comment:
please change this to local network params

Reply:
done